tools/pipelines/build-docs.yml

```diff
@@ -100,7 +100,7 @@ stages:
           submodules: false
           clean: true
 
-        - template: /tools/pipelines/templates/include-install-build-tools.yml
+        - template: /tools/pipelines/templates/include-install-build-tools.yml@self
```

Comment:
I take it recommended convention is to explicitly tag `@self`, despite that being the default?

Reply:
Yep. Seems reasonable to me, too.